yt_dlp/extractor/acfun.py

```diff
@@ -0,0 +1,192 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    float_or_none,
+    int_or_none,
+    traverse_obj,
+)
+
+
+class AcFunVideoIE(InfoExtractor):
+    _VALID_URL = r'(?x)https?://(?:www\.acfun\.cn/v/)ac(?P<id>[_\d]+)'
+
+    _TESTS = [{
+        'url': 'https://www.acfun.cn/v/ac35457073',
+        'info_dict': {
+            'id': '35283657',
+            'title': '【十五周年庆】文章区UP主祝AcFun生日快乐！',
+            'duration': 455.21,
+            'timestamp': 1655289827,
+        },
+        'params': {
+            # m3u8 download
+            'skip_download': True,
+        },
+        'skip': 'Geo-restricted to China',
```

Comment:
Is the whole site geo-restricted? If possible, add a non-restricted test

Reply:
ok